Marlin/Configuration_adv.h

```diff
@@ -930,6 +930,8 @@
   #if ENABLED(POWER_LOSS_RECOVERY)
     //#define POWER_LOSS_PIN         44 // Pin to detect power loss
     //#define POWER_LOSS_STATE     HIGH // State of pin indicating power loss
+    //#define POWER_LOSS_PULLUP
+    //#define POWER_LOSS_PULLDOWN
```

Comment:
I suppose it will always be `_PULLDOWN` when the `POWER_LOSS_STATE` is HIGH and `_PULLUP` when the `POWER_LOSS_STATE` is LOW. Hmm….

Reply:
good point, so perhaps 1 define for a pullupdown and the right one is done depending on the POWER_LOSS_STATE?